### ISSUE TITLE
Keyboard focus is moving to non-interactive element “End of chat history” note

### ISSUE THREAD

**Issue by Kirank915** [microsoft/BotFramework-WebChat#5021]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

Users who rely on keyboard will be impacted if Keyboard focus is moving to noninteractive element “End of chat history” note.

### What browsers and screen readers do this issue affect?

Others

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Focus management

### What is the public URL for the website?

https://airtuserexpdev.azurewebsites.net/Dashboard 

### How to reproduce the issue?

Step 1: Open the URL: [https://airtuserexpdev.azurewebsites.net/Dashboard](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fairtuserexpdev.azurewebsites.net%2fDashboard&data=05%7c01%7cv-shravyar%40microsoft.com%7c09aeb38517334354d48308dbeca14400%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c638363949604189146%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=X9RebjeUY64a0z9YqF59KH8oRldy59wbtdE%2fzj0CjvM%3D&reserved=0) in Edge Browser.

Step 2: AIRT Dashboard page will appear, Navigate to “Chat with Sage” chat box at the bottom of the page.

Step 3: Using TAB key navigate to bottom of the page.

Step 4: Verify whether keyboard Focus order is logical and moves on to noninteractive element “End of chat history” note or not.

### What do you expect?

Focus order should be logical and should not move to noninteractive element “End of chat history” note.
Expected Focus Order: Bot Message -> Text box.

### What actually happened?

Focus order is not logical and moves on to noninteractive element “End of chat history” note.
Focus Order: Bot Message -> "End of chat history" note -> Text box.

### Do you have any screenshots or recordings to repro the issue?

![MAS 2 4 3_Focus is moving on noninteractive element](https://github.com/microsoft/BotFramework-WebChat/assets/117266902/03b88174-4163-4bf3-82f0-265c223ed53d)

### MAS reference

https://microsoft.sharepoint.com/:w:/s/accessibility/EQnwpICBhyNEs80kSOr1v2IBQ1tiKQj6y90QKG1cRcwCEw?e=75BLFO

### WCAG reference

Understanding Success Criterion 2.4.3: Focus Order (w3.org)

**Comment by Kirank915** [comment]
Bug copied from Accenture team: [BUG 11645671](https://microsoftit.visualstudio.com/OneITVSO/_workitems/edit/11645671)

**Comment by Kirank915** [comment]
#A11yMAS;#A11ySev3;#MAS1.3.1;#HCL;#Accessibility;#FeatureBotframeworkwebchat-Apr23;

**Comment by compulim** [comment]
The "end of chat history" is by design and is a technical workaround for browser compatibility.
